src/AspNetCoreModuleV2/AspNetCore/aspnetcore_shim_config.cpp

```diff
@@ -13,7 +14,7 @@ ASPNETCORE_SHIM_CONFIG::Populate(
     IHttpApplication *pHttpApplication
 )
 {
-    STACK_STRU(strHostingModel, 12);
+    STRU                            strHostingModel;
```

Comment:
If someone puts an arbitrary value for the hosting model, we don't want to worry about buffer overflows.

Reply:
STACK_STRU would reallocate AFAIK